### ISSUE TITLE
From CustomCSSforFx_4.3.6 completely new appearance with same settings (of v_4.3.4)

### ISSUE THREAD

**Issue by Ransom77** [Aris-t2/CustomCSSforFx#595]
Hello

since years I manage with the following settings to make the FF look like Version 3:

![FF_113_ CustomCSSforFx_4 3 4_01 04 2023](https://github.com/Aris-t2/CustomCSSforFx/assets/66835201/14055290-7e44-4893-927e-f38089f49df4)

Disable:
@import "./config/general_variables.css";
@import "./config/color_variables.css";
@import "./css/buttons/buttons_on_navbar_classic_appearance.css";
@import "./css/buttons/icons_colorized.css";

Enable:
@import "./config/color_variables_classic-grey.css";
@import "./css/buttons/icons_custom_icons_fx3.css";
@import "./css/buttons/custom_backforward_large_ff3.css";
@import "./css/tabs/tabs_below_navigation_toolbar.css";
@import "./css/generalui/close_icon_firefox3.css";

Until CustomCSSforFx_4.3.4 everything was good. Today I changed to version 4.3.6, and now my FF (113.01) suddenly looks like this, even though I chose exactly the same settings (extra compared again).

![FF_113_ CustomCSSforFx_4 3 6_11 05 2023](https://github.com/Aris-t2/CustomCSSforFx/assets/66835201/725b3e89-7ce2-4476-85dd-a31c0c87c928)

Why is that?

Greetings
Ransom77

**Comment by Aris-t2** [comment]
Something must went wrong while you setup 4.3.6.
On your screenshot we can see, that no custom CSS is active.

**Comment by Aris-t2** [comment]
You also need to enable  `@import "./css/toolbars/menubar_color.css";`.

Looks like this for me, if I follow your config:

![image](https://github.com/Aris-t2/CustomCSSforFx/assets/6013569/e10f4ebc-9107-4386-b879-8c00b49df95d)

[CSS.zip](https://github.com/Aris-t2/CustomCSSforFx/files/11466296/CSS.zip)

**Comment by Ransom77** [comment]
Hello Aris,

many thanks for your answers.

You write:

> Something must went wrong while you setup 4.3.6.
> On your screenshot we can see, that no custom CSS is active.

I did nothing else (as usual) but copy the new (unzipped) files and the customized userChrome.css into the directory C:\Users\username\AppData\Roaming\Mozilla\Firefox\Profiles\xxxxxxxx.default\Chrome. What went wrong there, I can not understand.

I have now downloaded your (kindly) provided CSS.zip and copied it into the corresponding folder as indicated before – and lo and behold: everything is as before!

Thanks a lot, Aris! The new switch you pointed out I have noted (for future versions).

Greetings,
Ransom77
